service/observe/subscription.go

```diff
@@ -180,3 +218,18 @@ func (s *Subscriber) WatchLoop(watch WatchObjects) {
 		}
 	}
 }
+
+func (s *Subscriber) incrFailure(namespace, jobType string) {
```

Comment:
This function is self contained. Make it a package level function instead of a receiver to Subscriber. Same goes for the `incSuccess` and `initMetrics`.